specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,527 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRuleList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of Activity Log Alert rules could not be retrieved.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupName}/providers/Microsoft.AlertsManagement/tenantActivityLogAlerts": {
+      "get": {
+        "description": "Get a list of all Tenant Activity Log Alert rules in a management group.",
+        "operationId": "TenantActivityLogAlerts_ListByManagementGroup",
+        "x-ms-examples": {
+          "List Activity Log Alerts by management group": {
+            "$ref": "./examples/TenantActivityLogAlertRule_ListByManagementGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRuleList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of Activity Log Alert rules could not be retrieved.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureResource": {
+      "description": "An Azure resource object.",
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource Id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "The location of the resource. Since Azure Activity Log Alerts is a global service, the location of the rules should always be 'global'.",
+          "type": "string",
+          "default": "global",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
```

Comment:
(I'm actually a little confused about the value of a 'location' property in this case. Isn't 'global' not significantly different from having no location? Is it being used for routing somehow?)

Reply:
Activity Log Alerts Resource provider is a global service - therefore the expected location will always 'global' - and it will always be routed to the same traffic manager